CHANGELOG.md

```diff
@@ -1,208 +1,295 @@
+
+### Additions
+
+* CUDA 12.2.2 support
+* cuDNN 8.9.5 support
+* LLVM 17.0.3 support
+* Improved support for Windows and Visual Studio 2019 and 2022
+* More rocSPARSE support
+* ABI changes are shown in the 'C' ('Changed') column for CUDA, HIP, and ROC API
+
+### Known issues
+
+* [#837] Added a new function to call transformation type "additional non-const arg"
+* [#1014] Added a new function to call transformation type "replace argument with a const"
 
 ## HIPIFY for ROCm 5.7.0
-### Added
-- CUDA 12.2.0 support
-- cuDNN 8.9.2 support
-- LLVM 16.0.6 support
-- Initial rocSPARSE support
-- Initial CUDA2ROC documentation generation for rocBLAS, rocSPARSE, and MIOpen:
-  - in separate files: hipify-clang --md --doc-format=full --doc-roc=separate
-  - in a single file: hipify-clang --md --doc-format=full --doc-roc=joint
-- New options:
-  - --use-hip-data-types (Use 'hipDataType' instead of 'hipblasDatatype_t' or 'rocblas_datatype')
-  - --doc-roc=\<value\> (ROC documentation generation: 'skip' (default), 'separate', or 'joint'; the '--md' or '--csv' option must be specified)
-### Fixed
-- [#822] Add a new function call transformation type "additional const by value arg"
-- [#830] Add a new function call transformation type "move arg from place X to place Y"
+
+### Additions
+
+* CUDA 12.2.0 support
+* cuDNN 8.9.2 support
+* LLVM 16.0.6 support
+* Initial rocSPARSE support
+* Initial `CUDA2ROC` documentation generation for rocBLAS, rocSPARSE, and MIOpen:
+  * In separate files: `hipify-clang --md --doc-format=full --doc-roc=separate`
+  * In one file: `hipify-clang --md --doc-format=full --doc-roc=joint`
+* New options:
+  * `--use-hip-data-types` (Use 'hipDataType' instead of 'hipblasDatatype_t' or 'rocblas_datatype')
+  * `--doc-roc=\<value\>` (ROC documentation generation: `skip` (default), `separate`, and `joint`; the 
+    `--md` or `--csv` option must be included)
+
+### Known issues
+
+* [#822] Added a new function to call transformation type "additional const by value arg"
+* [#830] Added a new function to call transformation type "move arg from place X to place Y"
 
 ## HIPIFY for ROCm 5.6.0
-### Added
-- CUDA 12.1.0 support
-- cuDNN 8.8.1 support
-- LLVM 16.0.0 support
-- New options:
-  - --default-preprocessor (synonymous with '--skip-excluded-preprocessor-conditional-blocks')
-  - --no-undocumented-features
-  - --no-warnings-on-undocumented-features
-  - --versions
-### Fixed
-- Accessing half2 struct members (undocumented feature)
-- INSTALL to 'bin' subfolder
+
+### Additions
+
+* CUDA 12.1.0 support
+* cuDNN 8.8.1 support
+* LLVM 16.0.0 support
+* New options:
+  * `--default-preprocessor` (synonymous with `--skip-excluded-preprocessor-conditional-blocks`)
+  * `--no-undocumented-features`
+  * `--no-warnings-on-undocumented-features`
+  * `--versions`
+
+### Fixes
+
+* Accessing `half2 struct` members (undocumented feature)
+* Moved `INSTALL` to the `bin` subfolder
```

Comment:
INSTALL is a cmake operation, so it is better to reword to `Retarget INSTALL to bin subfolder`